src/main/java/org/fundacionjala/at15/pokemon/commands/create/CreatePokemon.java

```diff
@@ -9,27 +9,38 @@
 @Command(name = "pokemon", description = "Create a pokemon")
 public class CreatePokemon implements Callable<Integer> {
     private String idPokemonCreated;
+    private Pokemon newPokemon;
+    private String warningMessage = "";
 
-    @Option(names = { "-name", "-n" }, description = "pokemon name", defaultValue = "Pikachu")
+    @Option(names = { "-name", "-n" }, description = "pokemon name")
     private String pokemonName;
 
-    @Option(names = { "-hitPoints", "-hp" }, description = "pokemon Hit-Points", defaultValue = "100")
+    @Option(names = { "-hitPoints", "-hp" }, description = "pokemon Hit-Points")
     private int hitPoints;
 
     @Override
-    public Integer call() {
-        Pokemon newPokemon = new Pokemon(hitPoints, pokemonName);
-        this.idPokemonCreated = newPokemon.getId();
-        Writer.writeToJson(newPokemon);
-        System.out.println(
-                "Pokemon Created: \n"
-                + "Name: " + newPokemon.getPokemonName() +  "\n"
-                + "Hit points:" + newPokemon.getHitPoints().getCurrentHitPoints() + "\n"
-                + "ID: " + idPokemonCreated);
+    public Integer call() throws IncompleteArguments {
```

Comment:
I like it! 🤙🏼

Reply:
try/catch is an async promise. It will work but it could be better to think another way.

Reply:
I agree, we can review it in a meeting